### ISSUE TITLE
How to query oci object storage files

### ISSUE THREAD

**Issue by gassuncao-ccm** [ClickHouse/ClickHouse#70847]
### Question

Hello guys!

I'm starting to use clickhouse and have been facing some problems to query files from oracle object storage via s3 compability.

Im gonna let my tries bellow:

SELECT * FROM s3(

'https://<namespace>.compat.objectstorage.<region>.oraclecloud.com/<bucket\_name>/<path>/\*.parquet',  

<access\_key>,

<secret\_key>,

'Parquet'

);
The error I receive with that instruction:

Code: 499. DB::Exception: Received from localhost:9000. DB::Exception: Could not list objects in bucket <bucket_name> with prefix <path>, S3 exception: `SignatureDoesNotMatch`, message: 'The secret key required to complete authentication could not be found. The region must be specified if this is not the home region for the tenancy.': The table structure cannot be extracted from a Parquet format file. You can specify the structure manually. (S3_ERROR)

If i pass the full parquet path, i receive that error:

Code: 499. DB::Exception: Received from localhost:9000. DB::Exception: Failed to get object info: No response body.. HTTP response code: 403: while reading <full_path>: The table structure cannot be extracted from a Parquet format file. You can specify the structure manually. (S3_ERROR)

I use that keys in other scripts, so i really think they're ok.

I know its probably skill issue, so I'd appreciate some guidance. A example would be very helpfull!!

Thanks anyway!

**Comment by hanfei1991** [comment]
I think you were almost successful. According to `The table structure cannot be extracted from a Parquet format file. `, you can try 
```
SELECT * FROM s3(

'https://.compat.objectstorage..oraclecloud.com/<bucket_name>//*.parquet',

<access_key>,

<secret_key>,

'Parquet',
'a Int64, b String'

);
```

The fifth parameter means the structure of the table

**Comment by gassuncao-ccm** [comment]
Hi!

I found what was the problem.
My tenancy was created in a different region than the bucket that I was trying to query, gererating the following error:
 - The region must be specified if this is not the home region for the tenancy.'. (S3_ERROR) (version 24.9.2.42 (official build))

But even passing the bucket region on the URL, it did not recognized the endpoint (and passing the tenancy region, the bucket was not found).

So I created a bucket in the same region than the namespace and it worked.

Now I'm trying to find someway to query buckets in different regions than the tenancy.

Thanks for then help!
